diffkemp/simpll/FFI.cpp

```diff
@@ -0,0 +1,61 @@
+//===----------------- FFI.cpp - C interface for SimpLL -------------------===//
+//
+//       SimpLL - Program simplifier for analysis of semantic difference      //
+//
+// This file is published under Apache 2.0 license. See LICENSE for details.
+// Author: Tomas Glozar, tglozar@gmail.com
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains definitions of C functions used for interacting with
+/// DiffKemp.
+///
+//===----------------------------------------------------------------------===//
+
+#include "FFI.h"
+#include "Config.h"
+#include "Output.h"
+#include <cstring>
+
+extern "C" {
+void runSimpLL(const char *ModL,
```

Comment:
A large part of this function is common with `main` in `SimpLL.cpp`. I suggest to put what is common into a new function. It could prove useful if we change the analysis workflow in future (e.g. if post-processing is re-introduced). 

I'd put the function into `Transforms`. Actually, while looking into it now, the `Transforms` file should probably be renamed to something like `Analysis` or `MainAnalysis` (other suggestions?) since it would better express its purpose.

Reply:
Maybe `ModuleAnalysis` would be a good name - it makes it clear that it does more than `ModuleComparator` and that it operates on modules.